style.css

```diff
@@ -397,18 +331,166 @@ textarea::placeholder {
   height: 48px;
   border-radius: 4px;
   border: 1px solid #36b37f;
-  font-family: 'Inter', sans-serif;
   font-size: 17px;
   line-height: 24px;
   font-weight: 500;
   color: #fff;
 }
 
-footer {
-  padding-top: 5px;
+#personal-email {
+  font-size: 20px;
+  font-weight: 400;
+  line-height: 32px;
+  color: #42526e;
+  margin-top: 0;
+  margin-bottom: 0;
+  text-align: center;
+  padding-left: 10px;
+  padding-right: 0;
 }
 
 .b-action:hover {
   background: #008552;
   box-shadow: 0 8px 16px rgba(54, 179, 127, 0.24);
 }
+
+@media only screen and (max-width: 768px) {
+  .container {
+    background-image: url("./images/header-illsutration-mobile.svg");
+    background-repeat: no-repeat;
+    background-position: center top 60px;
+    background-size: contain;
+  }
+
```

Comment:
- [ ] Hello @AleMdta, great effort working on your project, but currently I notice you background images in the header, about and contact section is overlapping with the text and I would kindly suggest you work on them and make sure it looks like what we have in the Figma design. You have a good design for your mobile and I would appreciate if same fit can be achieved for the desktop design too. :+1: :+1: